content/building-blocks/store-and-document-your-data/document-data/documenting-new-data.md

```diff
@@ -11,7 +11,7 @@ aliases:
 
 ## Overview
 
-Ideally, your data description includes the *very elaborate questions* outlined in `Datasheets for datasets` by Gebru, Timnit, et al. (2018)[^1].
+Ideally, your data description includes the *very elaborate questions* outlined in Datasheets for datasets by Gebru, Timnit, et al. (2018)[^1].
```

Comment:
Improved overview section:

Ideally, your data description should embody the comprehensive principles outlined in the "Datasheets for Datasets" by Gebru, Timnit, et al. (2018)[^1^]. We highly recommend referring to the original paper for an in-depth exploration of the seven essential components crucial for meticulous dataset documentation.

In the following section, we have replicated these pivotal questions. It's advisable to incorporate them into a `readme.txt` file alongside your datasets. In the case of derived data, it may suffice to reference a relevant source code file and provide a comprehensive list of variables along with their operational definitions.



[^1^]: Add reference

Reply:
Looks great! I've just substituted the old version with this one and push the changes 🔝